Generate en of slack representative hand-over procedure (#1001)
utelecon/utelecon.github.io: src/pages/en/slack/workspace/index.mdx

#### Transfer of representatives
{:#take-over}

> In UTokyo Slack, who the representative (the person responsible for the management and operation) of each workspace is recorded, in addition to the information on owner/administrator roles stored in the system. At the time the workspace is created, the person who has submitted the application form is regarded as the representative. If you wish to transfer the representative to another person, proceed according to the following procedure.

Reviewer: ```suggestion
In UTokyo Slack, in addition to the system owner/administrator rights, we also manage who is responsible for the management and operation of each workspace. At the time the workspace is created, the person who has submitted the application form is regarded as the representative. If you wish to transfer the representative to another person, proceed according to the following procedure.
```

Reviewer 2: 大筋いいと思うんですが，
- rights → role
- we がUTokyo Slackの管理者であるとこれで読み取れるんでしたっけ
- we also ← system owner/administrator roleは我々が管理するともいえないような気がしています，alsoは抜いていいのでは

Author: えっと，

- はい
- 同じページ中でそういう用法で使ってるので，そこはよさそう
- そうしましょうか．owner roleの削除はアレですが，まあそこは本題じゃないので．
